.claude/scripts/flow-scorecard.sh

```
@@ -0,0 +1,266 @@
+  fi
+fi
+
+# ── Oldest open substantive issue per repo ───────────────────────
+if want age; then
+  echo ""
+  echo "── Oldest open substantive issue per repo (types: Feature, Bug, Security, Performance) ──"
+  substantive=$(load_array FLOW_SUBSTANTIVE_JSON bash -c '
+    for t in Feature Bug Security Performance; do
+      gh api "search/issues?q=org:devantler-tech+is:issue+is:open+type:$t&sort=created&order=asc&per_page=100" \
+        --paginate --jq ".items[] | {number, created_at, repository_url, type: \"$t\"}"
+    done | jq -s .')
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)